rbi/stdlib/psych.rbi

```diff
@@ -271,128 +271,99 @@ module Psych
+  # can be allowed by adding those classes to the +permitted_classes+ keyword argument.  They are
+  # additive.  For example, to allow Date deserialization:
   #
-  # ```ruby
-  # Psych.safe_load(yaml, permitted_classes: [Date])
-  # ```
+  #   Psych.safe_load(yaml, permitted_classes: [Date])
   #
-  # Now the [`Date`](https://docs.ruby-lang.org/en/2.7.0/Date.html) class can be
-  # loaded in addition to the classes listed above.
+  # Now the Date class can be loaded in addition to the classes listed above.
   #
-  # Aliases can be explicitly allowed by changing the `aliases` keyword
-  # argument. For example:
+  # Aliases can be explicitly allowed by changing the +aliases+ keyword argument.
+  # For example:
   #
-  # ```ruby
-  # x = []
-  # x << x
-  # yaml = Psych.dump x
-  # Psych.safe_load yaml               # => raises an exception
-  # Psych.safe_load yaml, aliases: true # => loads the aliases
-  # ```
+  #   x = []
+  #   x << x
+  #   yaml = Psych.dump x
+  #   Psych.safe_load yaml               # => raises an exception
+  #   Psych.safe_load yaml, aliases: true # => loads the aliases
   #
-  # A
-  # [`Psych::DisallowedClass`](https://docs.ruby-lang.org/en/2.7.0/Psych/DisallowedClass.html)
-  # exception will be raised if the yaml contains a class that isn't in the
-  # `permitted_classes` list.
+  # A Psych::DisallowedClass exception will be raised if the yaml contains a
+  # class that isn't in the +permitted_classes+ list.
   #
-  # A
-  # [`Psych::BadAlias`](https://docs.ruby-lang.org/en/2.7.0/Psych/BadAlias.html)
-  # exception will be raised if the yaml contains aliases but the `aliases`
-  # keyword argument is set to false.
+  # A Psych::BadAlias exception will be raised if the yaml contains aliases
+  # but the +aliases+ keyword argument is set to false.
   #
-  # `filename` will be used in the exception message if any exception is raised
+  # +filename+ will be used in the exception message if any exception is raised
   # while parsing.
   #
-  # When the optional `symbolize_names` keyword argument is set to a true value,
-  # returns symbols for keys in
-  # [`Hash`](https://docs.ruby-lang.org/en/2.7.0/Hash.html) objects (default:
-  # strings).
+  # When the optional +symbolize_names+ keyword argument is set to a
+  # true value, returns symbols for keys in Hash objects (default: strings).
+  #
+  #   Psych.safe_load("---\n foo: bar")                         # => {"foo"=>"bar"}
+  #   Psych.safe_load("---\n foo: bar", symbolize_names: true)  # => {:foo=>"bar"}
   #
-  # ```ruby
-  # Psych.safe_load("---\n foo: bar")                         # => {"foo"=>"bar"}
-  # Psych.safe_load("---\n foo: bar", symbolize_names: true)  # => {:foo=>"bar"}
-  # ```
   sig do
     params(
       yaml: T.any(String, StringIO, IO),
-      legacy_permitted_classes: Object,
-      legacy_permitted_symbols: Object,
-      legacy_aliases: Object,
-      legacy_filename: Object,
```

Comment:
Same for these. These should not be removed.

Reply:
reverted